src/azure-cli/azure/cli/command_modules/netappfiles/tests/latest/test_backup_policy_commands.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
-LOCATION = "eastus2euap"
+LOCATION = "eastus2"
```

Comment:
`eastus2euap` is not avaliable for the service now.

Reply:
reverted the change as the new location has permission errors, will leave it to service team to fix.